modules/node-shared/src/main/scala/org/tessellation/node/shared/domain/seedlist/SeedlistEntry.scala

```diff
@@ -75,19 +84,24 @@ object SeedlistEntry {
       row.values.size match {
         case 1 =>
           CellDecoder[PeerId].apply(row.values.head).map {
-            SeedlistEntry(_, none, none, none, none)
-          }
-        case 4 =>
-          decoder4Fields(row).map {
-            case (peerId, maybeIpAddress, maybeAlias, maybeBias) =>
-              SeedlistEntry(peerId, maybeIpAddress, maybeAlias, maybeBias, none)
+            SeedlistEntry(_, none, none, none, none, none)
           }
         case 5 =>
           decoder5Fields(row).map {
-            case (peerId, maybeIpAddress, maybeAlias, maybeBias, maybeOrdinalTimeline) =>
-              SeedlistEntry(peerId, maybeIpAddress, maybeAlias, maybeBias, maybeOrdinalTimeline)
+            case (peerId, maybeIpAddress, maybePort, maybeAlias, maybeBias) if maybeIpAddress.isDefined && maybePort.isDefined =>
```

Comment:
There is no need for additional conditional. Please see my snippet on slack

Reply:
I need to store the `Option`. If I did it like

```scala
            case (peerId, Some(ipAddress), Some(port), maybeAlias, maybeBias) =>
              SeedlistEntry(peerId, ipAddress.some, port.some, maybeAlias, maybeBias, none)
```

I would have to re-wrap the `ipAddress` and `port` in `Option`s. I thought that using a conditional was a clearer way to communicate my intention.

Reply:
Will this do what you want?
```suggestion
            case (peerId, maybeIpAddress@Some(_), maybePort@Some(_), maybeAlias, maybeBias) =>
              SeedlistEntry(peerId, maybeIpAddress, maybePort, maybeAlias, maybeBias, none)
```

Reply:
It does. Changed to:

```scala
          decoder5Fields(row).map {
            case (peerId, maybeIpAddress @ Some(_), maybePort @ Some(_), maybeAlias, maybeBias) =>
              SeedlistEntry(peerId, maybeIpAddress, maybePort, maybeAlias, maybeBias, none)
            case (peerId, _, _, maybeAlias, maybeBias) =>
              SeedlistEntry(peerId, none, none, maybeAlias, maybeBias, none)
```